src/App.tsx

```diff
@@ -1,9 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+const includesQueryText = (text: string, query: string) => (
+  text.toLowerCase().replace(/ /g, '')
+    .includes(query.toLowerCase().replace(/ /g, ''))
+);
+
 export const App: React.FC = () => {
+  const [query, search] = useState('');
+
+  const visibleMovies
+    = [...moviesFromServer].filter(({ title, description }) => (
```

Comment:
```suggestion
    = moviesFromServer.filter(({ title, description }) => (
```
filter returns new array